app/assets/stylesheets/base.scss

```diff
@@ -0,0 +1,61 @@
+a {
+  &.disabled {
+    color: $gray-ccc;
+    cursor: not-allowed;
+    text-decoration: none;
+  }
+}
+
+body {
+  font-family: 'Source Sans Pro', 'Arial Unicode MS', Helvetica, sans-serif;
+}
+
+a, a:hover, a:focus {
+  text-decoration: none;
+  border-bottom: 1px dotted $sul-link-color-border;
+}
+
+h1 {
+  font-family: 'dejavu_sansextralight';
+  color: $sul-h1-font-color;
+  padding: 10px 0;
+  letter-spacing: -0.05em;
+}
+
+h2 {
+  letter-spacing: 0;
+  line-height: 1.2em;
+  margin: 0.5em 0;
+  color: $cardinal-red;
+  border-bottom: 1px solid $sul-h2-border-color;
+}
+
+h3 {
+  font-weight: 400;
+  line-height: 1.3em;
+  margin: 0 0 0.4em;
+}
+
+dl {
+  margin-bottom: 15px;
+}
+
+dt {
+  color: $gray-41-percent;
+  font-weight: normal;
```

Comment:
Properties should be ordered color, font-size, font-weight, padding-right, padding-top, text-transform, width